modules/utils/docker/entrypoint_nats.sh

```diff
@@ -13,16 +13,26 @@ if [ "$MSVERSION" != "nats" ]; then
   fi
 else
 
+  #######################################
+  # BC needs to be on Place before this #
+  #######################################
+
+  # TODO: Identity from BC or HSM?
   if [ ! -f "/opt/identity" ]; then
       echo "generates identity id"
       generate_identity_id
   fi
 
+  #######################################
+  # Initialise default radios and IF    #
+  #######################################
+
   # Do not continue in case halow init has not finished
   while ps aux | grep [i]nit_halow > /dev/null; do
       sleep 1
   done
 
+  # TODO: remove as SLAAC and CBMA?
```

Comment:
Remove yes please: in general bridges are not even supposed to have an IP address 😉 In our case they already have an IPv6 Link-Local Address (fe80::...) anyway and when enabled, SLAAC is responsible for more addresses indeed.

Reply:
We need to wait CBMA to come in and then make other changes to support batman routing only.

Reply:
removed